include/net/net_context.h

```diff
@@ -878,6 +878,19 @@ int net_context_get_option(struct net_context *context,
 			   enum net_context_option option,
 			   void *value, size_t *len);
 
+/**
+ * @brief Find matching network context with given input values.
+ *
+ * @param family IP address family (AF_INET or AF_INET6 or AF_PACKET)
```

Comment:
As we are getting non IP protocols, better to say here `@param family Network protocol address family.`

Reply:
ok.